tests/additional_dictionary.txt

```diff
@@ -5,6 +5,7 @@ AddTransform
 Affine
 AffineTransform
 Another's
```

Comment:
What is the reason for adding additional words to the custom dictionary?

Reply:
As far as I could tell, these words were not included in the words in the dictionaries Windows used so spelling errors were getting flagged.  I couldn't find a way to install other dictionaries with the Windows version of pyenchant.  The only workaround I could find was to add these words in the custom dictionary.